test/e2e_test/e2e_test.py

```diff
@@ -0,0 +1,153 @@
+from datetime import date, datetime
+import e2e_utils
+import json
+from jsonschema import validate
+from math import nan
+import numpy as np
+import os
+import pickle
+import torch
+from typing import List, Tuple
+import subprocess
+import sys
+sys.path.append('./..')
```

Comment:
This should be after all the imports, right?

Reply:
This was to ensure the utils module can be imported. I am planning to refactor this test completely so I will get rid of that line